Lesoon 43 spring boot starter actuator

KaralchukArtem opened the pull request.

KaralchukArtem committed:
Update README.md

Commit:
feat: done html ...
task 1 -> create table
task 2 -> added css block

Commit:
feat: fix ...
task1 -> removed <h1> from the table

Commit:
feat: add ...
task1 -> add "wrapper"
task2 -> add "card-wrapper" vs bootstrap

Commit:
feat: JSTL completed

Commit:
feat: Postgresql done

Commit:
feat: Postgresql done

Commit:
Revert "feat: Postgresql done"

This reverts commit a8454be91875f05b80eebdfcb263468beee03556.

Commit:
feat: Postgresql done

Commit:
feat: done DB ...
INSERT , SELECT

Commit:
feat: done JDBC

Commit:
fix: changed method "get" to method "post" ...
ChangeLoginServlet, DeleteUserServlet.
Default INSERT in CreateUserServlet

Commit:
fix: done spring AOP

Commit:
fix: refactor Mapping

Commit:
feat: lesson 41 converted to SpringBoot

DHrytsuk commented on pom.xml:
```diff
@@ -0,0 +1,93 @@
+
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-dependencies</artifactId>
```
It should be spring-boot-starter-parent

DHrytsuk commented on pom.xml:
```diff
@@ -0,0 +1,93 @@
+            <version>42.7.3</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```
WE don't need this dependency, when using spring boot

DHrytsuk commented on pom.xml:
```diff
@@ -0,0 +1,93 @@
+            <artifactId>spring-context</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```
WE don't need this dependency, when using spring boot

DHrytsuk commented on src/main/java/lesson43/pack/PostgresDriverManager.java:
```diff
@@ -0,0 +1,35 @@
+
+public class PostgresDriverManager {
+    private static PostgresDriverManager instance;
+    private static String URL = "jdbc:postgresql://localhost:5432/tms";
```
Use these properties from application.properties file, not as hardcoded values

DHrytsuk commented on src/main/java/lesson43/service/UserService.java:
```diff
@@ -0,0 +1,77 @@
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+@Repository
```
```suggestion
@Service
```

DHrytsuk commented on src/main/java/lesson43/controller/UserController.java:
```diff
@@ -0,0 +1,60 @@
+    }
+
+    @PostMapping(path = "/create", consumes = {MediaType.APPLICATION_FORM_URLENCODED_VALUE})
+    public ModelAndView post(UserModel model) {
```
Rename methods to more user-friendly

Commit:
fix: ...
Removed unnecessary dependencies; pom.xml
The annotation was replaced by @Service; UserService
Use these properties from application.properties file
Rename methods by UserController
